src/markers/MarkerArrayClient.js

```diff
@@ -22,73 +22,85 @@
  *                         ROS3D.COLLADA_LOADER_2) -- defaults to ROS3D.COLLADA_LOADER_2
  */
 ROS3D.MarkerArrayClient = function(options) {
-  var that = this;
   options = options || {};
-  var ros = options.ros;
-  var topic = options.topic;
+  this.ros = options.ros;
+  this.topic = options.topic;
```

Comment:
I'm modified `this.topic` to become `this.topicName`. I think changing any of the `subscribe()` signatures is too drastic of a break especially since we don't even have older versions in NPM yet for people to pin to if they don't keep up with changes or have time to re-work. Totally cool to make larger changes once something is in npm. Memory cost of duplicate strings is negligible when compared to the overall footprint of webgl rendering.